nautobot/ui/package.json

```diff
@@ -22,6 +22,7 @@
         "@testing-library/user-event": "^13.5.0",
         "axios": "^1.2.1",
         "framer-motion": "^9.0.3",
+        "nautobot-ui": "github:nautobot/nautobot-ui",
```

Comment:
Added inadvertently? We already have this at line 15 above.

Reply:
So when I do `npm add https://github.com/nautobot/nautobot-ui.git` or `npm add nautobot/nautobot-ui` it adds this line instead of the one above. Maybe `"@nautobot/nautobot-ui": "nautobot/nautobot-ui",` has come backward capability handling?

Reply:
Fixed @glennmatthews